base/linalg/symmetric.jl

```diff
@@ -64,6 +64,8 @@ issymmetric{T<:Complex,S}(A::Hermitian{T,S}) = all(imag(A.data) .== 0)
 issymmetric(A::Symmetric) = true
 transpose(A::Symmetric) = A
 ctranspose{T<:Real}(A::Symmetric{T}) = A
+isposdef(A::Symmetric) = isposdef(A.data)
```

Comment:
Since `Symmetric` is a symmetric *view* of the matrix it is not correct to call `isposdef` on the `data` field. In many cases `data` will be a triangular matrix. Instead, you should probably expand [this signature](https://github.com/JuliaLang/julia/blob/beac4d38421e5cfa3419ef36ebf6d48629684927/base/sparse/cholmod.jl#L1526) to allow for `Symmetric` and `Hermitian` sparse matrices.